### DISCUSSION TITLE
Email Proxy

### DISCUSSION THREAD

**Discussion by TheMinecraftGuyGuru** [NginxProxyManager/nginx-proxy-manager#1828] [Ideas]
I was curious what it would take to implement the email proxy (as found on [nginx.com here](https://docs.nginx.com/nginx/admin-guide/mail-proxy/mail-proxy/)) into the GUI. I currently run Portainer, with the NginxProxyManager being the only one with external ports, which then goes to the internet, and I’m looking to add iRedMail to my stack of services, and thought that proxying it would be a great idea.

**Comment by FrAllard** [comment]
If the nginx integrated to NPM has the "mail" component compiled in it, you can do it manually using custom configuration.

You would need to put your custom configuration in a file custom/root.conf I believe.
